src/System.Runtime.Extensions/src/System/Environment.Unix.cs

```diff
@@ -18,7 +18,7 @@ public static partial class Environment
     {
         internal static readonly bool IsMac = Interop.Sys.GetUnixName() == "OSX";
         private static Func<string, IEnumerable<string>> s_fileReadLines;
-        private static Action<string> s_directoryCreateDirectory;
+        private static Func<string,object> s_directoryCreateDirectory;
```

Comment:
Spacing after comma

Reply:
Noticed after the PR created. updating right away.